src/utils/get-current-time.ts

```diff
@@ -0,0 +1,5 @@
+import { getOverrideTimestamp } from "../blockchain/process-block";
+
+export function getCurrentTime(): number {
+  return  getOverrideTimestamp() || Date.now() / 1000;
```

Comment:
Bringing system time to blockchain time seems like it could cause unforeseen problems? I wonder if this would be better served getting the head block time?

Reply:
Yea that's a good point. Will change.